### PR TITLE
Fix for windows zoom (ScaleFactor)

### PR DESCRIPTION
**Pull request by Sickjuicy**
Fixes the ScaleFactor with windows zoom.

Adds the scalefactor, for one monitor or multiple, testet on 2 monitors and one, on my laptop and pc also sandbox.

Laptop zoom: 150% and than 100%
PC two monitor:

zoom: 100%
zoom: 150%

References to  #868

### CODE REVIEW

**Review by Sickjuicy — COMMENTED** [review]
Adds the scalefactor, for one monitor or multiple, testet on 2 monitors and one, on my laptop and pc  also sandbox.

Laptop zoom: 150% and than 100%
PC two monitor: 
1. zoom: 100%
2. zoom: 150%

File: plugins/startup-FN/back.js
Comment (JellyBrick) [inline plugins/startup-FN/back.js:1]: Why did you add this plugin?

File: plugins/startup-FN/front.js
Comment (JellyBrick) [inline plugins/startup-FN/front.js:1]: Same as the comment above.

**Review by JellyBrick — CHANGES_REQUESTED** [review]

File: plugins/startup-FN/back.js
Comment (Sickjuicy) [reply plugins/startup-FN/back.js:1]: I have not seen the function in the shortcuts plugin, the other plugin is there for when I start the PC that you can use the function keys directly without the need to move the program into focus. I have noticed in general in the installer version the function keys often do not work but in the standalone version. I have the plugins in the after no also deleted again, unfortunately, this is very confusing in github just, should I reopen the post? 
PS: english is not my main language

Comment (Sickjuicy) [reply plugins/startup-FN/back.js:1]: I deleted the plugin.

File: index.js [hunk 1]
```diff
@@ -177,7 +177,28 @@ function createMainWindow() {
...
+	const size = config.get("window-size");
+	const position = config.get("window-position");
+	
+	if (size && size.width && size.height) {
+	  const scaledSize = {
+		width: size.width / scaleFactor,
+		height: size.height / scaleFactor,
+	  };
+	  win.setSize(scaledSize.width, scaledSize.height);
+	}
```
Comment (JellyBrick) [inline index.js]:
```suggestion
	if (size && size.width && size.height) {
		const scaledSize = {
			width: size.width / scaleFactor,
			height: size.height / scaleFactor,
		};
		win.setSize(scaledSize.width, scaledSize.height);
	}
```

File: index.js [hunk 2]
```diff
@@ -177,7 +177,28 @@ function createMainWindow() {
...
+	
+	if (position && position.x && position.y) {
+	  const scaledPosition = {
+		x: position.x / scaleFactor,
+		y: position.y / scaleFactor,
+	  };
+	  win.setPosition(scaledPosition.x, scaledPosition.y);
+	}
```
Comment (JellyBrick) [inline index.js]:
```suggestion
	if (position && position.x && position.y) {
		const scaledPosition = {
			x: position.x / scaleFactor,
			y: position.y / scaleFactor,
		};
		win.setPosition(scaledPosition.x, scaledPosition.y);
	}
```

**Review by JellyBrick — CHANGES_REQUESTED** [review]

**Review by JellyBrick — APPROVED** [review]

**Comment by th-ch** [comment]
Thanks, ported in https://github.com/th-ch/youtube-music/commit/d36fb592d0ad12152e2d46429bbb8bb13519e63e in TS!

**Pull request closed without merging**